modules/nodes-containers-sysctls-unsafe.adoc

```diff
@@ -26,27 +26,28 @@ containers, resource shortage, or breaking a node.
 
 .Procedure
 
-. Add a label to the machine config pool where the containers where containers
-with the unsafe sysctls will run:
+. List existing MachineConfig objects for your {product-title} cluster to decide how to label your machine config:
 +
 [source,terminal]
 ----
-$ oc edit machineconfigpool worker
+$ oc get machineconfigpool
 ----
 +
-[source,yaml]
+.Example output
+[source,terminal]
 ----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfigPool
-metadata:
-  creationTimestamp: 2019-02-08T14:52:39Z
-  generation: 1
-  labels:
-    custom-kubelet: sysctl <1>
+NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master   rendered-master-bfb92f0cd1684e54d8e234ab7423cc96   True      False      False      3              3                   3                     0                      42m
+worker   rendered-worker-21b6cb9a0f8919c88caf39db80ac1fce   True      False      False      3              3                   3                     0                      42m
 ----
-<1> Add a `key: pair` label.
 
-. Create a `KubeletConfig` custom resource (CR):
+. Add a label to the machine config pool where the containers with the unsafe sysctls will run:
++
+[source,terminal]
+----
+$ oc label machineconfigpool worker custom-kubelet=sysctl
+----
+. Create a YAML file `sysctl-example-safe-unsafe.yaml` that defines a `KubeletConfig` custom resource (CR):
```

Comment:
Name of yaml is sysctl-example-safe-unsafe.yaml however in the next step we apply set-sysctl-worker.yaml